src/services/review-stages.js

```diff
@@ -0,0 +1,101 @@
+// Dependencies
+const httpStatusCode = require('@generics/http-status')
+const reviewStageQueries = require('@database/queries/reviewStage')
+const responses = require('@helpers/responses')
+
+module.exports = class reviewStagesHelper {
+	/**
+	 * update review stage
+	 * @method
+	 * @name update
+	 * @param {Object} bodyData - review stage body data.
+	 * @returns {JSON} - update review stage response.
+	 */
+
+	static async update(id, bodyData, organization_id) {
+		try {
+			bodyData.organization_id = organization_id
+			//validate resource type
+			const validResourceTypes = process.env.RESOURCE_TYPES.split(',')
+			if (!validResourceTypes.includes(bodyData.resource_type)) {
+				return responses.failureResponse({
+					message: `resource_type ${bodyData.resource_type} is not a valid`,
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			//update review stage
+			const [updateCount, updatedReviewStage] = await reviewStageQueries.updateOne(
+				{ id: id, organization_id: organization_id },
+				bodyData,
```

Comment:
can we omit id from the body data so that the id doesn't get updated. Or should we go for the blacklisting? @priyanka-TL

Reply:
@adithyadinesh0412 that is planned in the security audit

Reply:
Got it. @priyanka-TL